websites/T/Tempest Fansub/presence.ts

```diff
@@ -0,0 +1,136 @@
+const presence = new Presence({
+		clientId: "1047102386478534727",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+presence.on("UpdateData", async () => {
+	const { pathname } = window.location,
+		presenceData: PresenceData = {
+			startTimestamp: browsingTimestamp,
+			largeImageKey: "https://i.imgur.com/MVWiX8p.png",
+		},
+		_host = location.host;
```

Comment:
Destructure `document.location` for this instead

Reply:
i need both

Reply:
For?

Reply:
this website have 2 domains first manga.tempest second normal tempest, both on different topics but one site